keyboards/mechlovin/kanu/info.json

```
@@ -0,0 +1,230 @@
+                {"label":"K4A (B4,C7)", "x":10, "y":4, "w":1.25},
+                {"label":"K4B (B4,D0)", "x":11.25, "y":4, "w":1.25},
+                {"label":"K4C (B4,D1)", "x":13, "y":4},
+                {"label":"K4D (B4,D2)", "x":14, "y":4},
+                {"label":"K4E (B4,D3)", "x":15, "y":4}
+            ]
+        },        
+        "LAYOUT_iso": {
+            "layout": [
+                {"label":"K00 (B0,B5)", "x":0, "y":0},
+                {"label":"K01 (B0,B6)", "x":1, "y":0},
+                {"label":"K02 (B0,B7)", "x":2, "y":0},
+                {"label":"K03 (B0,C0)", "x":3, "y":0},
+                {"label":"K04 (B0,C1)", "x":4, "y":0},
+                {"label":"K05 (B0,C2)", "x":5, "y":0},
+                {"label":"K06 (B0,C3)", "x":6, "y":0},
+                {"label":"K07 (B0,C4)", "x":7, "y":0},
+                {"label":"K08 (B0,C5)", "x":8, "y":0},
+                {"label":"K09 (B0,C6)", "x":9, "y":0},
+                {"label":"K0A (B0,C7)", "x":10, "y":0},
+                {"label":"K0B (B0,D0)", "x":11, "y":0},
+                {"label":"K0C (B0,D1)", "x":12, "y":0},
+                {"label":"K0D (B0,D2)", "x":13, "y":0},
+                {"label":"K0E (B0,D3)", "x":14, "y":0},
+                {"label":"K49 (B4,C6)", "x":15, "y":0},
+                {"label":"K10 (B1,B5)", "x":0, "y":1, "w":1.5},
+                {"label":"K11 (B1,B6)", "x":1.5, "y":1},
+                {"label":"K12 (B1,B7)", "x":2.5, "y":1},
+                {"label":"K13 (B1,C0)", "x":3.5, "y":1},
+                {"label":"K14 (B1,C1)", "x":4.5, "y":1},
+                {"label":"K15 (B1,C2)", "x":5.5, "y":1},
+                {"label":"K16 (B1,C3)", "x":6.5, "y":1},
+                {"label":"K17 (B1,C4)", "x":7.5, "y":1},
+                {"label":"K18 (B1,C5)", "x":8.5, "y":1},
+                {"label":"K19 (B1,C6)", "x":9.5, "y":1},
+                {"label":"K1A (B1,C7)", "x":10.5, "y":1},
+                {"label":"K1B (B1,D0)", "x":11.5, "y":1},
+                {"label":"K1C (B1,D1)", "x":12.5, "y":1},
+                {"label":"K2D (B2,D2)", "x":13.75, "y":1, "w":1.25, "h":2},
+                {"label":"K1E (B1,D3)", "x":15, "y":1},
+                {"label":"K20 (B2,B5)", "x":0, "y":2, "w":1.75},
+                {"label":"K21 (B2,B6)", "x":1.75, "y":2},
+                {"label":"K22 (B2,B7)", "x":2.75, "y":2},
+                {"label":"K23 (B2,C0)", "x":3.75, "y":2},
+                {"label":"K24 (B2,C1)", "x":4.75, "y":2},
+                {"label":"K25 (B2,C2)", "x":5.75, "y":2},
+                {"label":"K26 (B2,C3)", "x":6.75, "y":2},
+                {"label":"K27 (B2,C4)", "x":7.75, "y":2},
+                {"label":"K28 (B2,C5)", "x":8.75, "y":2},
+                {"label":"K29 (B2,C6)", "x":9.75, "y":2},
+                {"label":"K2A (B2,C7)", "x":10.75, "y":2},
+                {"label":"K2B (B2,D0)", "x":11.75, "y":2},
+                {"label":"K2C (B2,D1)", "x":12.75, "y":2},
```

`info.json` needs to use the same order that a `keymap.c` would use.

```suggestion
                {"label":"K1E (B1,D3)", "x":15, "y":1},
                {"label":"K20 (B2,B5)", "x":0, "y":2, "w":1.75},
                {"label":"K21 (B2,B6)", "x":1.75, "y":2},
                {"label":"K22 (B2,B7)", "x":2.75, "y":2},
                {"label":"K23 (B2,C0)", "x":3.75, "y":2},
                {"label":"K24 (B2,C1)", "x":4.75, "y":2},
                {"label":"K25 (B2,C2)", "x":5.75, "y":2},
                {"label":"K26 (B2,C3)", "x":6.75, "y":2},
                {"label":"K27 (B2,C4)", "x":7.75, "y":2},
                {"label":"K28 (B2,C5)", "x":8.75, "y":2},
                {"label":"K29 (B2,C6)", "x":9.75, "y":2},
                {"label":"K2A (B2,C7)", "x":10.75, "y":2},
                {"label":"K2B (B2,D0)", "x":11.75, "y":2},
                {"label":"K2C (B2,D1)", "x":12.75, "y":2},
                {"label":"K2D (B2,D2)", "x":13.75, "y":1, "w":1.25, "h":2},
```